README.markdown

```diff
@@ -144,6 +144,18 @@ npm install --global @sourcemeta/jsonschema
 pip install sourcemeta-jsonschema
 ```
 
+### From mise
+
+```sh
+mise use jsonschema
+```
+
+### From aqua
+
+```sh
+aqua g -i sourcemeta/jsonschema
```

Comment:
Are you sure this works? I tried installing `aqua` from Homebrew and running this same command, but didn't work for me. Like above, maybe also worth linking to Aqua's homepage and the manifests that they have created for this tool, if any?

Finally, can we use the long form of the options instead of `g` and `-i`? That might better help readers understand

Reply:
Yeah, makes sense. Maybe let's add the other one (and the NPM cleanup) and we try to research more about Aqua before adding it?